fhttp/http_test.go

```diff
@@ -941,6 +941,24 @@ func TestChangeMaxPayloadSize(t *testing.T) {
 	}
 }
 
+func TestAddHTTPS(t *testing.T) {
+	var tests = []struct {
+		input    string
+		expected string
+	}{
+		{"www.google.com", "https://www.google.com"},
+		{"http://www.google.com", "https://www.google.com"},
+		{"https://www.google.com", "https://www.google.com"},
+	}
+
+	for _, test := range tests {
+		output := AddHTTPS(test.input)
+		if output != test.expected {
+			t.Errorf("%s was expected but %s is received ", test.expected, output)
```

Comment:
go convention is to print actual first, before expected (I don't think it's very good or important but so far it's been consistently applied)